controllers/templatesync/template_sync.go

```diff
@@ -690,6 +695,7 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 			// Handle cluster scoped objects
 			if isClusterScoped {
 				addFinalizer = true
+				createdGkConstraint = true
```

Comment:
It'd be nice to not assume that any cluster scoped policy template is Gatekeeper.